### PR TITLE
Add FAQ item about using kubectl debug node

### PR DESCRIPTION
**Pull request by max-rocket-internet**
**Issue number:** https://github.com/bottlerocket-os/bottlerocket/issues/4011

Closes https://github.com/bottlerocket-os/bottlerocket/issues/4011

**Description of changes:** Add item to FAQ about how to use `kubectl debug node`

**Terms of contribution:**

By submitting this pull request, I confirm that my contribution is made under
the terms of the licenses outlined in the LICENSE-SUMMARY file.

### CODE REVIEW

**Comment by taraspos** [comment]
It seems like `sysadmin` profile was added in 1.30, so probably it worth to mention in the doc as well:

- https://github.com/kubernetes/kubernetes/pull/119200
- https://github.com/kubernetes/kubernetes/blob/c8a51aaffc94da6e7a66d8984890d3111e361a54/CHANGELOG/CHANGELOG-1.30.md?plain=1#L498

**Comment by max-rocket-internet** [comment]
> It seems like sysadmin profile was added in 1.30, so probably it worth to mention in the doc as well

That's a great idea @taraspos! If only I'd thought of that. Oh wait, I did 😅

**Comment by taraspos** [comment]
Oh, sorry. Not sure how I missed that :D

**Comment by taraspos** [comment]
However, I see that it mentions `kubectl` only:

> Use a `kubectl` version of 1.30 or later to ensure that the `sysadmin` profile is available

Would it work if `kubectl` is 1.30, but cluster/kubelet itself is 1.29?

**Review by ginglis13 — APPROVED** [review]
LGTM, thanks!

**Comment by max-rocket-internet** [comment]
Thanks. I don't have access to merge so one of @ginglis13 or @taraspos will need to do it 🙂

**Review by KCSesh — APPROVED** [review]

File: content/en/faqitems/1_3-access-how-do-i-use-kubetl-debug.markdown [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+The [debug node](https://kubernetes.io/docs/tasks/debug/debug-cluster/kubectl-node-debug/) option from `kubectl` can be used to gain access to a Bottlerocket node as follows:
+
+```console
+kubectl debug node/<node name> -it --image=ubuntu --profile=sysadmin
+```
+
+This is a useful alternative to SSH or SSM. Use a `kubectl` version of 1.30 or later to ensure that the `sysadmin` profile is available. Once the debug pod is started and you have a shell, you can run `chroot /host apiclient exec admin bash` to access the admin container (if enabled) or `chroot /host <command>` to run commands from the Bottlerocket OS.
+
+The Pod created is very priviliged so ensure it is removed after use.
```
Comment (KCSesh) [inline content/en/faqitems/1_3-access-how-do-i-use-kubetl-debug.markdown:14]: Nit: `priviliged` is misspelled -> `privileged`

Also we can drop the word `very`

Comment (max-rocket-internet) [reply content/en/faqitems/1_3-access-how-do-i-use-kubetl-debug.markdown]: Fixed

**Review by larvacea — APPROVED** [review]
Thank you! TIL, thanks to this.

**Comment by max-rocket-internet** [comment]
Can we merge this already @larvacea @ginglis13 @taraspos?

**Pull request merged**